quil-py/quil/expression/__init__.pyi

```diff
@@ -63,17 +63,17 @@ class Expression:
     @staticmethod
     def new_pi() -> "Expression": ...
     @staticmethod
-    def from_address(memory_reference: MemoryReference) -> "Expression": ...
+    def from_address(inner: MemoryReference) -> "Expression": ...
```

Comment:
Per https://github.com/rigetti/rigetti-pyo3/blob/b43e780532fb7b2adf0e69e4957aa7fce935c4c9/src/wrappers.rs#L583 looks like `inner` is the right name 👍

Reply:
Yup! One of the nice things about updating pyo3 is that the `__text_signature__` property is now derived automatically from the Rust function signature (or the `#[pyo3(signature = (...))]` attribute). `stubtest` checks __text_signature__ and flags any hints that don't align with it, meaning we now get `stubtest` failures when a signature uses the wrong argument names or types whereas before it really only failed if a function was missing. That's what prompted all these fixes in the type hints.